angular/projects/admin-nrpti/src/app/mines/mines-record-add/mines-record-add.component.html

```diff
@@ -46,7 +46,9 @@
           </label>
           <select name="typeCode" id="typeCode" formControlName="typeCode" class="form-control">
             <option *ngFor="let typeCode of permitTypes" [ngValue]="typeCode">
-              {{ typeCode === 'OGP' ? 'Original Permit' : 'Permit Amendment' }}
+              {{
```

Comment:
I prefer to avoid nested ternary operators. I find they can be confusing to reason about and prefer the clarity of something along the following lines. Not a necessary change though. You'd have to pull it out into a function.

`
const acceptedTypeCodes = ['OGP', 'ALG']

if (acceptedTypeCodes.includes(typeCode)) return 'Permit Amendment';

if (typeCode === 'OGP') return 'Original Permit';

if (typeCode === 'ALG') return 'Amalgamated Permit';
`